src/pywink/api.py

```diff
@@ -101,6 +101,23 @@ def get_sirens():
     return get_devices(device_types.SIREN)
 
 
+def get_subscription_key():
+    arequest_url = "{}/users/me/wink_devices".format(WinkApiInterface.BASE_URL)
+    response = requests.get(arequest_url, headers=API_HEADERS)
+    if response.status_code == 200:
+        response_dict = response.json()
+        first_device = response_dict.get('data')[0]
+        if "subscription" in first_device:
+            return first_device.get("subscription").get("pubnub").get("subscribe_key")
+        else:
+            return None
+
+    if response.status_code == 401:
```

Comment:
This status code checking is duplicated in `get_devices`

You could create a new function called `wink_api_fetch` which handles all of these status codes and either raises or returns the `response_dict`.  Then you could replace `request.get(arequest_url...` with `wink_api_fetch`.

`wink_api_fetch` would be easily testable.  The remainder of `get_subscription_key` and `get_devices` would be so minimal that it probably wouldn't even affect test coverage.